components/Tabs/examples/TabBarIndicatorTemplateExample.swift

```diff
@@ -0,0 +1,119 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+import UIKit
+import CoreGraphics
+
+import MaterialComponents.MaterialAppBar
+import MaterialComponents.MaterialPalettes
+import MaterialComponents.MaterialTabs
+
+class TabBarIndicatorTemplateExample: UIViewController {
+
+  // MARK: Properties
+  var alignment: MDCTabBarAlignment {
+    get {
+      return tabBar.alignment
+    }
+    set(newAlignment) {
+      tabBar.setAlignment(newAlignment, animated: true)
+    }
+  }
+
+  lazy var alignmentButton: MDCRaisedButton = self.setupAlignmentButton()
+
+  lazy var appBar: MDCAppBar = self.setupAppBar()
+
+  lazy var tabBar: MDCTabBar = {
+    let tabBar = MDCTabBar()
+    tabBar.alignment = .justified
+
+    tabBar.items = [
+      UITabBarItem(title: "Fly", image: nil, tag:0),
+      UITabBarItem(title: "Sleep", image: nil, tag:0),
+      UITabBarItem(title: "Eat", image: nil, tag:0),
+    ]
+
+    let blue = MDCPalette.blue.tint500
+    tabBar.tintColor = blue
+    tabBar.inkColor = blue.withAlphaComponent(0.15)
+
+    tabBar.barTintColor = UIColor.white
+    tabBar.itemAppearance = .titles
+    tabBar.selectedItemTintColor = UIColor.black.withAlphaComponent(0.87)
+    tabBar.unselectedItemTintColor = UIColor.black.withAlphaComponent(0.38)
```

Comment:
If possible, can we rely on the catalog themers for color here? Do we need to override the colors specifically for this example?

Reply:
Done. Though it's not perfect. See https://github.com/material-components/material-components-ios/issues/2408/

Reply:
Ack.